src/github.ts

```diff
@@ -109,6 +109,7 @@ interface GraphQLRelease {
   url: string;
   description: string;
   isDraft: boolean;
+  isPrerelease: boolean;
```

Comment:
What is this value used for? from the tests it looks like this feature createss pre-releases, but it didn't seem to do any filtering of results based on this variable, or return it to the user (I might just be missing where it's used).

Reply:
Adding to the comment above, it's currently not used. It seems the `isDraft` of `GraphQLRelease` and `draft` of `GitHubRelease` is not used either. Maybe a cleanup for those in a separate PR?

Reply:
I wanted to make it consistent with `isDraft`. I'm gonna remove `isPrerelease` from `GraphQLRelease` as well as `prerelease` from `GitHubRelease` as they aren't used downstream. Do you think the `isDraft` and `draft` could be also cleaned up? it's also not used anywhere.